test/test_pipeline/test_classification.py

```diff
@@ -388,7 +386,7 @@ def test_get_hyperparameter_search_space(self):
 
         # The four parameters which are always active are classifier,
```

Comment:
Could you please update this docstring?

Reply:
Yes. Just wondering if the word "parameters" is the most appropriate here. Isn't "components" better?

Reply:
Yes, that would be better.

Reply:
Done.